compile_test.go

```diff
@@ -25,7 +25,7 @@ func TestEval(t *testing.T) {
 			"f": "(x: int) -> int",
 			"y": "42",
 		}
-		scope.Range(func(k string, v values.Value) {
+		scope.LocalRange(func(k string, v values.Value) {
```

Comment:
what does this fix?

Reply:
It gets `compile_test.go` to pass. Previously the test would range over the entire prelude, erroring when it found something other than `f` or `y`.